snuba/migrations/migrate.py

```diff
@@ -3,14 +3,27 @@
 from clickhouse_driver import Client
 from typing import MutableSequence
 
-from snuba.datasets.dataset import Dataset
 from snuba.datasets.schemas import Schema
 from snuba.datasets.schemas.tables import TableSchema
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_storage
 from snuba.migrations.parse_schema import get_local_schema
 
 
 logger = logging.getLogger("snuba.migrate")
 
+STORAGES_TO_MIGRATE = [
```

Comment:
Are these not taken from the list in the factory because you need to state the order in which they have to be migrated ?

Reply:
Yes. This part is certainly temporary until we move to a migration system that will define the order. But it defines the order of storages to be migrated, as well as omits the ones that should not be migrated (QUERYLOG and GROUPS)